protocols/collective_aggregation_protocol.go

```diff
@@ -201,14 +197,19 @@ func (p *CollectiveAggregationProtocol) Dispatch() error {
 
 // Announce forwarding down the tree.
 func (p *CollectiveAggregationProtocol) aggregationAnnouncementPhase() error {
+	timeout, err := time.ParseDuration(os.Getenv("MEDCO_TIMEOUT"))
```

Comment:
It should be enough to put this in a`func init()` at the beginning of the file `constants.go`. Then you can remove it from all the methods you're calling it...

Reply:
done abbe277